### DISCUSSION TITLE
Disabling transition on colorscheme toggle for burger component

### DISCUSSION THREAD

**Discussion by Sajarin-M** [mantinedev/mantine#3297] [Q&A] [answered]
Hi there! I was wondering if anyone could help me with a problem I am having with the `Burger` component. I have set a `transitionDuration` prop for it, but I am trying to find a way to avoid this transition when the colorscheme is toggled. Is there a way to do this that anyone is aware of? I would greatly appreciate any suggestions or ideas. Thank you in advance for your help!

![ezgif-1-2f3ed79ce4](https://user-images.githubusercontent.com/68892162/210964430-194c5dcc-794f-43c5-89c6-cf48dbc99dcb.gif)

**Comment by rtivital** [comment, marked as answer]
You can add utility class with the following css `.no-transitions * { transition: none !important }`, then subscribe to color scheme changes and add this class to body element, then remove it with setTimeout.
